anago

```diff
@@ -742,6 +743,23 @@ get_build_candidate () {
                                 $PARENT_BRANCH \
    || return 1
 
+  # When creating alphas, make sure that X.X.0 has been created first or warn.
```

Comment:
Let's construct some messaging that's concise but provides enough context to make an informed decision with this information.